tests/logger_test.go

```diff
@@ -0,0 +1,81 @@
+package logger_test
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"strings"
+	"testing"
+	"fmt"
+
+	"github.com/SENG-499-Company2-B01/Backend/logger"
+)
+
+func TestLogger(t *testing.T) {
```

Comment:
I would separate these tests into their own for cleanliness. Ideally a test will just test 1 thing (ex: Info Message)

Reply:
fixed 👍